lib/src/model/cose_error_code.dart

```diff
@@ -0,0 +1,12 @@
+enum CoseErrorCode {
+  none,
```

Comment:
Capitalize enum values

Reply:
That would break the current release on pub.dev